.gitignore

```diff
@@ -1 +1,8 @@
 .env
+terraform/creds.json
+terraform/.terraform
+terraform/.terraform.lock.hcl
+terraform/.terraform.tfstate.lock.info
+terraform/*.tfvars
+terraform/*.tfstate
+terraform/*.backup
```

Comment:
How about separating to a `.gitignore` specific to the terraform directory?

Reply:
I do not see the point. I'd still have the `.gitignore` at the root for the `.env`. This would just add an unnecessary file IMO.

Reply:
Since this is going to be a smaller repo, then it is not so much of an issue to have a larger `.gitignore` at root